### PR TITLE
Testing advantage

### PR DESCRIPTION
**Pull request by Captagonal**
# Your Name (real name not username)
Sawyer KoKo
### What Subsystem?
Pose Estimation for Sim driving and a better auto align(Auto Align still needs to be written)
### What Type?
 - [ ] Updated Code
 - [x] New Code
   - task #42 / #40 
### Have you tested the Code?
 - [X] Yes on the Simulator, but not on the bot
 - [ ] No
## Explain your code (What it does and how it does it)

I'm using the Swerve Pose Estimator and putting the pose into a field2d for visualization, I also fixed some bugs in ClimberLevelCommand so that it no longer kills the code. Attached is a video of the robot Driving in the Simulator 

I also fixed the workflow so that the check no longer fails

##### One of these People must review your code
@Captagonal @jsmmr2 @KateK13 @PalladiumSpider

https://github.com/first-team-93-new-apple-corp/Robot2024/assets/146396002/1439298e-1ac3-44c9-a916-f3b7a1c53295

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/ElevatorSubsystem.java
Comment (jsmmr2) [inline src/main/java/frc/robot/subsystems/ElevatorSubsystem.java:1]: Is there a reason you moved the definitions to global space? This was the only way I could get the robot to not crash.
@Captagonal

**Review by jsmmr2 — APPROVED** [review]
Looks good.

File: src/main/java/frc/robot/subsystems/ElevatorSubsystem.java
Comment (Captagonal) [reply src/main/java/frc/robot/subsystems/ElevatorSubsystem.java:1]: That’s what made it so that the check inside of GitHub no longer fails

Comment (Captagonal) [reply src/main/java/frc/robot/subsystems/ElevatorSubsystem.java:1]: Ignore my reply I misread what you said

**Pull request merged**